geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -140,7 +359,19 @@ app.DrawController = function($scope, ngeoDecorateInteraction) {
  * @private
  */
 app.DrawController.prototype.onDrawEnd_ = function(event) {
-  event.feature.set('name', 'element ' + (++this.featureSeq_));
+  var feature = event.feature;
+  feature.set('name', 'element ' + (++this.featureSeq_));
+  feature.set('__editable__', true);
+  feature.set('__style__', {
+    color: '#ed1c24',
+    opacity: 0.2,
+    size: feature.getGeometry().getType() == ol.geom.GeometryType.POINT ?
+        10 : 1.25,
+    shape: 'circle',
+    rotation: 0,
+    lineDash: 'plain'
+  });
```

Comment:
See https://github.com/Geoportail-Luxembourg/geoportailv3/pull/833/files for the list of feature properties expected by the mymaps web service.